WalletWasabi.Tests/RegressionTests/BackendTests.cs

```diff
@@ -177,14 +177,31 @@ public async Task StatusRequestTestAsync()
 
 		(_, IRPCClient rpc, _, _, _, _, Backend.Global global) = await Common.InitializeTestEnvironmentAsync(RegTestFixture, 1);
 
-		var indexBuilderService = global.IndexBuilderService;
+		var segwitTaprootIndexBuilderService = global.SegwitTaprootIndexBuilderService;
+		var taprootIndexBuilderService = global.TaprootIndexBuilderService;
+		if (segwitTaprootIndexBuilderService is null || taprootIndexBuilderService is null)
+		{
+			throw new InvalidOperationException("Index builders can't be null.");
+		}
+
 		try
 		{
-			indexBuilderService.Synchronize();
+			segwitTaprootIndexBuilderService.Synchronize();
+			taprootIndexBuilderService.Synchronize();
 			// Test initial synchronization.
 			var times = 0;
 			uint256 firstHash = await rpc.GetBlockHashAsync(0);
-			while (indexBuilderService.GetFilterLinesExcluding(firstHash, 101, out _).filters.Count() != 101)
+			while (segwitTaprootIndexBuilderService.GetFilterLinesExcluding(firstHash, 101, out _).filters.Count() != 101)
+			{
+				if (times > 500) // 30 sec
+				{
+					throw new TimeoutException($"{nameof(IndexBuilderService)} test timed out.");
```

Comment:
Shouldn't this one be:

```suggestion
					throw new TimeoutException($"{nameof(global.SegwitTaprootIndexBuilderService)} test timed out.");
```

If yes then line 208 should be:
`throw new TimeoutException($"{nameof(global.TaprootIndexBuilderService)} test timed out.");`

Reply:
Honestly I don't know what it should be. If you say so, then make a PR to my branch and I'll trust you and merge it.

Reply:
@Szpoti What do you think?

Reply:
It doesn't matter. It is a test and if one day it fails we will know the exact line where the exception was thrown.